lib/ansible/modules/cloud/vmware/vmware_cluster_facts.py

```diff
@@ -62,6 +69,16 @@
     cluster_name: DC0_C0
   delegate_to: localhost
   register: cluster_facts
+
+- name: Gather facts from datacenter about specific cluster with tags
+  vmware_cluster_facts:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    cluster_name: DC0_C0
+    show_tag: True
```

Comment:
Could you add a functional test to cover the new parameter?

Reply:
OK.